added comparing images

kbeker opened the pull request.

kbeker committed:
added comparing images

kbeker committed:
better image recognition

cameel commented:
Please don't store test data in the repo. Only code should be committed.

Also, there's a bit of code that looks like it was copied from somewhere. It's fine in a branch, as a starting point, but it should not be merged into master in that state.

kbeker committed:
added function to findAll

kbeker commented:
Ok, I will correct it tomorrow. Thanks

kbeker committed:
add .png .jpg to gitignore

kbeker committed:
add print to script and return variables; delete show plot

kbeker committed:
add histogram comparing, changed searching function

As a parametrs now we need to give image and size of small image what we
want to cut of big image.

kbeker committed:
add script for comparing images with random crop window

kbeker committed:
add check_all_pixels script
